### ISSUE TITLE
Channel semantics

### ISSUE THREAD

**Issue by fridex** [gofed/docs#2]
According to schema:
https://github.com/gofed/docs/blob/master/schemas/golang-project-exported-api.json#L113

channel could be "1", "2" or "3". This is confusing, since there is no meaning for defined values. I would suggest to define it according to Golang specification:
https://golang.org/ref/spec#Channel_types

e.g. "send", "receive" and "bidirectional".

**Comment by ingvagabund** [comment]
Agree. At the time it was "just" for comparison. There was no intent to provide the API. Now it is.
